packages/manager/modules/telecom-spare/src/phones/order/phones-order.html

```diff
@@ -0,0 +1,212 @@
+                                    <oui-spinner></oui-spinner>
+                                </ovh-contact>
+                            </div>
+                            <div>
+                                <h3 data-translate="phones_order_shipping_mode_title"></h3>
+                                <tuc-shipping-mode-selection data-ng-model="$ctrl.order.shipping.mode"
+                                    data-tuc-shipping-mode-selection-relay="$ctrl.order.shipping.relay"
+                                    data-tuc-shipping-mode-selection-options="$ctrl.order.shipping.options">
+                                </tuc-shipping-mode-selection>
+                            </div>
+                        </div>
+                        <div class="col-sm-6">
+                            <h2 class="widget-presentation-title" data-translate="phones_order_phone_title"></h2>
+                            <div>
+                                <h3 data-translate="phones_order_select_title"></h3>
+                                <oui-select name="brand"
+                                            data-model="$ctrl.order.brand"
+                                            data-items="$ctrl.brands"
+                                            placeholder="{{ ::'phones_order_place_holder' | translate}}"
+                                            data-match="name">
+                                </oui-select>
+                                <div>
+                                    <label data-translate="phones_order_select_quantity"></label>
+                                    <oui-numeric data-model="$ctrl.quantity" data-max="50"></oui-numeric>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <button type="button" class="oui-button oui-button_primary oui-button_icon-right float-right mt-2"
+                        data-ng-disabled="$ctrl.isDisabled()"
+                        data-ng-click="$ctrl.changeStep('summary')">
+                        <span data-translate="phones_order_order_next"></span>
+                        <span class="oui-icon oui-icon_circle oui-icon-chevron-right" aria-hidden="true"></span>
+                    </button>
+                </section>
+
+                <!-- SUMMARY -->
+                <section class="widget-presentation" data-ng-if="$ctrl.orderStep === 'summary'">
+                    <h2 class="widget-presentation-title" data-translate="phones_order_summary_title"></h2>
+
+                    <div class="text-center" data-ng-if="$ctrl.isStepLoading">
+                        <oui-spinner></oui-spinner>
+                    </div>
+
+                    <div data-ng-if="!$ctrl.isStepLoading">
+
+                        <!-- PRICE SUMMARY TABLE -->
+                        <h3 data-translate="phones_order_summary"></h3>
+                        <div class="oui-datagrid-responsive-container">
```

Comment:
Seems like this could be turned into a `oui-datagrid` component

Reply:
no, I already answered for this feedback, I use the style, it's efficient.

Reply:
But there is a difference between using the style and the component; the component add an abstraction layer to help us to display a datagrid, it's more evolutive and more easy to maintain. In this context, the component is the best solution.

Reply:
But this table is only to display the summary of the order of the customer for the spare.

Reply:
If every table that is 'only doing something' is written differently, then we can't keep them up to date when we update the datagrid :)

Reply:
in the start of the PR, I made a simple table to represents the summary (like in the order for telephony accessories or phones). You asked me to use a oui-datagrid or less the style. I used the style. 
Please, could you try to understand what I want to do and call me if you want more explanations.